lib/oauth.js

```diff
@@ -110,8 +112,12 @@ function requestHandler (environment, server, state, verifier) {
     await new Promise(resolve => server.stop(resolve));
     debug(`stopped http server listening on ${PORT}`);
 
-    if (params.state !== state) {
-      throw new Error('An error occurred during authentication');
+    debug(`state expected:${state} actual:${params.state}`);
+
+    if (!params.state) {
+      console.log(chalk.yellow('OAuth state is missing in the callback. Ignoring.'));
```

Comment:
```suggestion
     // Warn rather than erroring out if OAuth state is missing (#77)
      console.log(chalk.yellow('OAuth state is missing in the callback. Ignoring.'));
```